Documentation/AdditionalFunctionality/Jobs.md

```diff
@@ -117,7 +117,7 @@ Besides emitting events, `Job`s can also listen to changes of other [ObservedObj
 This will also trigger the `run` method of a `Job`.
 To differentiate between the scheduled execution and the execution of an `ObservableObject`, the projected Boolean value `changed` of the property wrapper can be used. Projected values of property wrappers are accessed using the `$` prefix operator.
 
-The _RegisterHandler_ is responsible to count every access to its path and store this information in _VisitorObservedObject_. Using `Environment(value, keyPath)` in the `configuration` stored property, which takes as arguments a key path and the corresponding value, we can declare the _VisitorObservedObject_ as a Singleton. This allows us to use the `ObservableObject` in the web service with the property wrappers `@Environment` and `@ObservedObject` and the key path. The `Job` will send a weekly push notification based on the total number of visitors and also when someone accessed the _RegisterHandler_ by listening to changes to the _VisitorObservedObject_. To differentiate between these two cases we use the projected value `$visitors.changed` to check if the `Job` was triggered by being scheduled or by the _VisitorObservedObject_.
+The _RegisterHandler_ is responsible to count every access to its path and store this information in _VisitorObservedObject_. Using `Environment(value, keyPath)` in the `configuration` stored property, which takes as arguments a key path and the corresponding value, we can declare the _VisitorObservedObject_ as a Singleton. This allows us to use the `ObservableObject` in the web service with the property wrappers `@Environment` and `@ObservedObject` and the key path. The `Job` will send a weekly push notification based on the total number of visitors and also when someone accessed the _RegisterHandler_ by listening to changes to the _VisitorObservedObject_. To differentiate between these two cases we use the property `changed` of `@Published`  to check if the `Job` was triggered by being scheduled or by the _VisitorObservedObject_.
```

Comment:
I think this should be "the property `changed` of `ObservedObject`" to be consistent with your implementation, shouldn't it?

Reply:
Good catch. Thank you for pointing this out!